riva/clients/nmt/riva_nmt_streaming_s2s_client.cc

```diff
@@ -71,6 +71,9 @@ DEFINE_bool(
     "Whether to use SSL credentials or not. If ssl_cert is specified, "
     "this is assumed to be true");
 DEFINE_string(metadata, "", "Comma separated key-value pair(s) of metadata to be sent to server");
+DEFINE_string(rate, "", "Speech rate for TTS output");
+DEFINE_string(pitch, "", "Speech pitch for TTS output");
+DEFINE_string(volume, "", "Speech volume for TTS output");
```

Comment:
could you name these arguments as `tts_rate`, `tts_pitch`, `tts_volume` for clarity?

Reply:
on another thought, should we call them as `tts_prosody_rate` to be clear? we don't want to get it confused with `tts_sample_rate` for example. Thoughts? or any other better term?

Reply:
changed to `tts_prosody_rate`